cli/cli.py

```diff
@@ -482,11 +566,14 @@ def decorator(func: Callable) -> Callable:
                 in option_type_info.items():
             arg = f'--{opt_name}'
             if opt_main_type == list:
-                click.option(arg, multiple=True,
+                if opt_additional_types[0] == desbordante.data_types.Table:
+                    click.option(arg, type=(str, str, bool),
```

Comment:
[This](https://click.palletsprojects.com/en/8.1.x/parameters/#implementing-custom-types) should help avoid the workaround  (the elif will become redundant as well)